cheatsheets/PHP_Configuration_Cheat_Sheet.md

```diff
@@ -111,3 +107,11 @@ report_memleaks         = On
 track_errors            = Off
 html_errors             = Off
 ```
+
+## Suhosin (only usable in 5.X branch and should not be used in prod now as 5.X is EOL)
+
+Consider using [Suhosin](https://suhosin.org/) if you want to patch many custom security flaws in various parts of PHP. 
+
+## Snuggleupagus (7.X Series, beta product use with caution)
```

Comment:
Is this stable enough to be recommending? The suhosin link in here makes it pretty clear you shouldn't be using it for anything important.

Reply:
The product is out since 2016 i have seen traces online where providers are implementing it and this is the only option at this point besides php.ini tweaking if you want to control PHP. not recommending it will be a dis-service. Hence recommended with caveat.